src/datashare/azext_datashare/azext_metadata.json

```diff
@@ -1,4 +1,4 @@
-{
-    "azext.isExperimental": true,
-    "azext.minCliCoreVersion": "2.3.1"
+{
+    "azext.isExperimental": true,
```

Comment:
Did you remove the experimental tag manually? If you modified the readme.md and re-generate extension code, the flag here would be removed too.

Reply:
I reinstated the experimental tag in commands.py file. Never changed anything on this file since the first commit.

Reply:
Can you confirm that if service is GA? Do you want to add preview tag? If you remove experimental tag without adding other tag, that means this extension is stable. And all interfaces change would be considered as breaking change later.

Reply:
The service is GA. I'm not removing the experimental tag (as Kai told me that it is suggested to us experimental tag if it is the first release)